package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "governance-portal-v2",
   "license": "AGPL-3.0-only",
-  "version": "0.7.3",
+  "version": "0.7.3-test",
```

Comment:
this should be removed, i changed it to not collide with the other caches (since we use the version number on the cache string name)